styles/colors-carbon.less

```diff
@@ -186,8 +186,8 @@
 @agate-slider-knob-bg-image:  linear-gradient(to bottom, transparent, fade(#333, 50%));
```

Comment:
Let's replace #333 with @agate-darker-gray.

Reply:
I found #333 more `4 times in colors-carbon.less` and `2 times in colors-silicon-day.less`.